Formula/tailscale.rb

```diff
@@ -0,0 +1,35 @@
+class Tailscale < Formula
+  desc "Easiest, most secure way to use WireGuard and 2FA"
+  homepage "https://tailscale.com"
+  url "https://github.com/tailscale/tailscale.git",
+      tag:      "v1.16.2",
+      revision: "b56ba20549aedc0e56d09dca2552ca62421b2cbd"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    vars=Utils.safe_popen_read("./build_dist.sh", "shellvars")
+    ldflags = %W[
+      -s -w
+      -X tailscale.com/version.Long=#{vars.match(/VERSION_LONG=\"(.*)\"/)[1]}
+      -X tailscale.com/version.Short=#{vars.match(/VERSION_SHORT=\"(.*)\"/)[1]}
```

Comment:
Would be good to have something that will fail hard if/when this ever doesn't match as I think this will just produce an empty version

Reply:
That's what the version check will (partially) do